tests/test_timeouts.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from unittest import TestCase
+from unittest.mock import patch
+
+from rsconnect.exception import RSConnectException
+from rsconnect.timeouts import get_timeout
+
+
+class GetTimeoutTestCase(TestCase):
```

Comment:
trivial note - you could add a test for a "0" timeout value here just because its right on the conditional boundary.